terraform/aws/modules/compute/main.tf

```diff
@@ -104,29 +104,42 @@ resource "aws_ecs_service" "km_ecs_service" {
     container_port   = 80
   }
   network_configuration {
-    assign_public_ip = true
+    assign_public_ip = false
     subnets          = var.private_subnet
-    security_groups  = [ var.elb_sg ]
+    security_groups  = [var.elb_sg]
   }
   tags = merge(var.default_tags, {
   })
 }
 
 resource "aws_cloudwatch_log_group" "km_log_group" {
```

Comment:
**Tenable.cs remediation**
 
**[HIGH]: Ensure KMS CMK is used to encrypt AWS CloudWatch Log Group**
 
**Vulnerability**
KMS CMK is not used to encrypt AWS CloudWatch Log Group which may lead to sensitive data exposure.
 
**Remediation**
AWS CloudWatch Log Group can be encrypted using AWS KMS.
 In AWS Console - 
 1. Create an AWS KMS customer managed key. 
 2. Set the necessary permissions on the customer managed key. 
 3. Associate a log group with a customer managed key.

source: tenablecs-